src/Tizen.Content.Download/Tizen.Content.Download/Request.cs

```diff
@@ -843,6 +844,37 @@ private void UnregisterProgressChangedEvent()
                 DownloadErrorFactory.ThrowException(ret, "Unsetting ProgressChanged callback failed");
             }
         }
+
+        /// <summary>
+        /// Enabled state of the cache feature.
+        /// </summary>
+        /// <since_tizen> 7.5 </since_tizen>
+        /// <privilege>http://tizen.org/privilege/download</privilege>
+        /// <exception cref="ArgumentException">Thrown when it is failed due to an invalid parameter.</exception>
```

Comment:
see my comment for CacheManager.MaxCacheSize, it applies here too. But this one is public.

Reply:
yes. updated.